acra/configuring-maintaining/logging/output.md

```diff
@@ -0,0 +1,59 @@
+---
+weight: 2
+title: Logging output
+bookCollapseSection: true
+---
+
+## Logging output
+
+The default output is `stderr`. Examples of output:
+
+#### Plaintext:
```

Comment:
Sectioning could be improved – here and in other files – since you're no longer constrained by one-page docs.

The first, "title" section is typically always `<h1>` (`#` in Markdown), and then next levels are used as needed.

Markdown for the "old" doc server often used arbitrary nesting to get the section headers colored "nicely", disregarding proper nesting. I'd argue that new doc server should use proper nesting instead. That is, if your parent section is `#` then the child section should be `##` — not `###` or `####` "because it looks better". If you think that you need to use a different section level then the section title is probably too long, or you are missing an appropriate section in between.

For example, here you could use an extra section to group log formats:

```markdown
# Logging output

## Format examples

### Plaintext

### CEF

### JSON
```

Reply:
Agree with sectioning you mentioned above, but in my opinion additional group log formats section will be sort of redundant here because hugo server by default  provide an ability to switch between sections, like this:
![Screenshot from 2021-07-08 17-59-22](https://user-images.githubusercontent.com/24659475/124945789-25eb9d80-e017-11eb-8d67-f7f9195c296e.png)
the section displayed on the page.

Reply:
> additional group log formats section will be sort of redundant here because hugo server by default provide an ability to switch between sections

Makes sense 👍 Given that it's a single page dedicated to log formats.

(Though I still think you should drop the colons and use `##`.)